hack/testing/init-log-stack

```diff
@@ -33,6 +33,7 @@ openshift_logging_image_prefix=$imageprefix
 openshift_logging_use_ops=${ENABLE_OPS_CLUSTER:-False}
 openshift_logging_fluentd_use_journal=${USE_JOURNAL:-}
 openshift_logging_fluentd_journal_read_from_head=${JOURNAL_READ_FROM_HEAD:-False}
+openshift_logging_es_log_appenders=['file','console']
```

Comment:
We should always be against using console when logs from the console get fed back into the Elasticsearch instance from which they were emitted.